py-packages/openverse-attribution/src/openverse_attribution/attribution.py

```diff
@@ -0,0 +1,62 @@
+import re
+
+from openverse_attribution.license import License
+
+
+def get_attribution_text(
+    license_slug: str,
+    title: str | None = None,
+    creator: str | None = None,
+    license_version: str | None = None,
+    license_url: str | bool | None = None,
+):
+    """
+    Get the attribution text for a media item. This function only renders the
+    attribution in plain-text format for the English language.
+    Note that this is not a perfect attribution as it does not include
+    hyperlinks for the work or the creator. Also see the CC `wiki`_ to learn
+    best practices for attribution.
+
+    .. _wiki: https://wiki.creativecommons.org/wiki/Best_practices_for_attribution
+
+    To remove the sentence for viewing the legal text, set the ``license_url``
+    parameter to ``False``.
+
+    :param title: the name of the work, if known
+    :param creator: the name of the work's creator, if known
+    :param license_slug:
+    :param license_version: the version of the license, if known
+    :param license_url: the URL to the license, to override the default
+    :return: the plain-text English language attribution
+    """
+
+    lic = License(license_slug)
+
+    title = f'"{title}"' if title else "This work"
+
+    attribution_template = "{title} {creator} {marked-licensed} {license}. {view-legal}"
```

Comment:
Nit: This could probably be a module-level constant.

Reply:
That's true, but since this is very implementation specific, I did not want it to be exposed to the outside world and it's much cleaner imo if the module only exports one item: function `get_attribution_text`.

Reply:
That's understandable, but we could still make it module-level without necessarily exposing it (e.g. use an underscore as the prefix, like `_ATTRIBUTION_TEMPLATE`). That way it's clear for the library developers without necessarily being an exported attribute.

Reply:
My comment in this thread is also relevant here: https://github.com/WordPress/openverse/pull/4107/files#r1574945150

We can sort these details out later, they shouldn't block this PR, the implementation details here are out of scope for the project and don't have lasting implications we can't sort out later.